lab_6_pipeline/pipeline.py

```diff
@@ -144,16 +260,57 @@ def __init__(self, corpus_manager: CorpusManager):
         """
         Initializes MorphologicalAnalysisPipeline
         """
+        self._corpus = corpus_manager
+        self._mystem = Mystem()
+        self._converter_path = Path(__file__).parent/'data'/'mystem_tags_mapping.json'
+        self._converter = MystemTagConverter(self._converter_path)
 
     def _process(self, text: str) -> List[ConlluSentence]:
         """
         Returns the text representation as the list of ConlluSentence
         """
+        sentences = split_by_sentence(text)
+        conllu_sent = []
+        punct = '!"#$%&()*+,-/:;<=>?@[\]^_`{|}~'
+        for sent_id, sentence in enumerate(sentences):
+            conllu_tokens = []
+            result = [i for i in self._mystem.analyze(sentence) if i['text'].strip()
+                      not in punct and i['text'].strip()]
+            for token_id, token in enumerate(result, start=1):
+                if token.get('analysis'):
+                    lemma = token['analysis'][0]['lex']
+                    pos = re.match(r'[A-Z]+', token['analysis'][0]['gr']).group()
+                    ud_pos = self._converter.convert_pos(pos)
```

Comment:
you need to make ALL processing of mystem output in `mystemconverter`

Reply:
done